util/generate_family_demand.py

```diff
@@ -0,0 +1,20 @@
+# This code generates tables with the demand of each instance family, using the TOTAL_demand file.
+
+import pandas as pd
+
+total_demand = pd.read_csv('TOTAL_demand.csv')
+
```

Comment:
after we add a function that receives a path, we should add a doc to this function. This doc must indicate the format of the file (it is a csv file, I guess, so we should add the name of the expected columns